examples/gno.land/r/demo/bug/bug_test.gno

```diff
@@ -0,0 +1,12 @@
+package bug
+
+import (
+	"testing"
+)
+
+func TestSetNumber(t *testing.T) {
+	defer reset()
```

Comment:
I noticed you don't have the panic if you wrap the call in a function literal :

```go
defer func() {
  reset()
}()
```

Reply:
thanks, the two cases deserves dedicated `gnovm/tests` files.